src/lib/util.js

```diff
@@ -486,7 +494,21 @@ export function loadScripts(urls) {
         }
     });
 
-    return Promise.all(promises);
+    return Promise.all(promises)
+        .then(() => {
+            if (disableRequireJS) {
+                window.define = define;
+                window.require = require;
+                window.requirejs = requirejs;
+            }
+        })
+        .catch(() => {
```

Comment:
Added a catch in case we fail to load the script, we should still relinquish control over require.

Reply:
Too bad we can't use `finally` yet... :(

Chrome 63/Firefox 58 is probably too new: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally